source/deimos/openssl/tls1.di

```diff
@@ -160,6 +160,9 @@ nothrow:
 
 enum TLS1_ALLOW_EXPERIMENTAL_CIPHERSUITES = 0;
 
+static if (OPENSSL_VERSION_AT_LEAST(1, 1, 1))
+    enum TLS1_3_VERSION = 0x0304;
+
```

Comment:
Looks like those are in a new file since v3.0.0:
https://github.com/openssl/openssl/commit/085e3cecbdadde10e93abfb4dbd3e865aeed58d6

Reply:
Done.